modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tsurugitachi' AND ID = 23042;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Turquoise Earring' AND ID = 22029;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Turquoise Earring' AND ID = 22060;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Turquoise Ring' AND ID = 22545;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Turquoise Ring' AND ID = 22580;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Turtle Bangles' AND ID = 52508;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Twicer' AND ID = 11548;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Two-Hand. Sword (desynth)' AND ID = 19970;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Two-Handed Sword' AND ID = 12012;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Unicorn Cap' AND ID = 54556;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Unicorn Harness' AND ID = 55005;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Unicorn Leggings' AND ID = 54544;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Unicorn Mittens' AND ID = 54531;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Unicorn Subligar' AND ID = 54512;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vagabond\'s Gloves' AND ID = 40010;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Velocity Bow' AND ID = 3539;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Verdun' AND ID = 24575;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Crab Shell' AND ID = 52002;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Femur' AND ID = 50507;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Scorpion Claw' AND ID = 52025;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Sheep Leather' AND ID = 40007;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Sheep Leather' AND ID = 40008;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Vivio Wyvern Scale' AND ID = 53510;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Volt Gun' AND ID = 22536;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wailing Shell' AND ID = 50010;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Walahra Burner' AND ID = 24519;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Cloth' AND ID = 34051;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34044;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34046;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34048;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='War Aketon' AND ID = 34033;
```

Comment:
War Aketon Set has Zilart entries as far back as 2004, with confirmed recipes in COP:
http://web.archive.org/web/20050331232525/https://ffxi.allakhazam.com/db/item.html?fitem=3366